opensrp-chw-hps/src/main/java/org/smartregister/chw/hps/interactor/BaseHpsServiceVisitInteractor.java

```diff
@@ -136,6 +210,19 @@ private void evaluateReferralServices(Map<String, List<VisitDetail>> details) th
         actionList.put(context.getString(R.string.hps_referral_services), action);
     }
 
+    private void evaluateHpsRemarks(Map<String, List<VisitDetail>> details) throws BaseHpsVisitAction.ValidationException {
+
+        HpsRemarksActionHelper actionHelper = new HpsRemarksActionHelper(mContext, memberObject);
+        BaseHpsVisitAction action = getBuilder(context.getString(R.string.hps_remarks))
+                .withOptional(false)
```

Comment:
this should be `.withOptional(true)` since the comments section is optional and the user should be able to save even without filling any comments